docs/maintain-guides-how-to-join-council.md

```diff
@@ -57,6 +57,5 @@ left column under the heading "Members."
 ![e](assets/council/member.png)
 
 Congratulations! Now you are able to participate on the council by making motions or vetoing
-proposals. It's a good idea to now [set a nick](https://guide.kusama.network/en/latest/try/nicks/)
```

Comment:
These two docs files `docs/maintain-guides-how-to-join-council.md` and `docs/maintain-guides-secure-validator.md` aren't actually changing from the already published ones, this PR hadn't pulled it from the master.